src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/administration/CreateRoomScheduleService.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.administration
+
+import jakarta.persistence.EntityNotFoundException
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.LocationUsage
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.LocationUsage.SCHEDULE
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request.CreateRoomScheduleRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository.LocationAttributeRepository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.ExternalUser
+import java.util.UUID
+
+@Service
+class CreateRoomScheduleService(
```

Comment:
I know there is no unit test for this.  We do however have a basic integration test.